src/showingpreviously/cinemas/dominion.py

```diff
@@ -0,0 +1,72 @@
+import datetime
+from bs4 import BeautifulSoup
+
+import showingpreviously.requests as requests
+from showingpreviously.model import ChainArchiver, CinemaArchiverException, Chain, Cinema, Screen, Film, Showing
+from showingpreviously.consts import UK_TIMEZONE, UNKNOWN_FILM_YEAR
+
+
+FILM_SCHEDULE_URL = 'https://www.dominioncinema.co.uk/schedule/'
+FILM_NAME_IGNORES = []
+
+CHAIN = Chain('Dominion')
+CINEMA = Cinema('Dominion', UK_TIMEZONE)
+
+
+def get_response(url: str) -> requests.Response:
+    r = requests.get(url)
+    if r.status_code != 200:
+        raise CinemaArchiverException(f'Got status code {r.status_code} when fetching URL {url}')
+    return r
+
+
+def format_film_name(film_name: str) -> (str, dict[str, any]):
+    attributes = {}
+    # convert to lowercase for easier processing
+    film_name = film_name.lower()
+    # check for live showings
+    if '(live)' in film_name:
```

Comment:
Although the Dominion does use `(Live)` to denote live screenings, it also has it in non-parentheses, for example `Royal Ballet: The Sleeping Beauty - Live`. Could maybe add a check if the title ends with ` - Live` as well?